rules/collections/annotations/java.yaml

```diff
@@ -29,14 +29,8 @@ collections:
       - "RemoteServiceRelativePath"
     tags:
 
-  - id: Collections.Annotation.Retrofit
```

Comment:
Retrofit is one of the ways to implement JAX-RS, others include Jersey, RESTEasy, RetroFit.